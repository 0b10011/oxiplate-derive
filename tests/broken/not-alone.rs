use oxiplate_derive::Oxiplate;

#[derive(Oxiplate)]
#[oxiplate_inline = r##"{% if ! %}"##]
struct NotAlone {}

fn main() {
    let data = NotAlone {};

    print!("{}", data);
}
