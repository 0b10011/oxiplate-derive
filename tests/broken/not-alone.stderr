error: Expected an expression after prefix operator
 --> tests/broken/not-alone.rs:4:32
  |
4 | #[oxiplate_inline = r##"{% if ! %}"##]
  |                                ^
